### PR TITLE
delete redundant variable and const

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
redundant err defined 
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/utils/runners.go [hunk 1]
```diff
@@ -993,7 +993,6 @@ func MakePodSpec() v1.PodSpec {
 }
 
 func makeCreatePod(client clientset.Interface, namespace string, podTemplate *v1.Pod) error {
-	var err error
 	for attempt := 0; attempt < retries; attempt++ {
 		if _, err := client.Core().Pods(namespace).Create(podTemplate); err == nil {
```
Comment (liggitt) [inline test/utils/runners.go]: this actually looks like it was trying to save the error inside the loop for printing in the final return, but accidentally shadowing the err var

File: test/utils/runners.go [hunk 2]
```diff
@@ -993,7 +993,6 @@ func MakePodSpec() v1.PodSpec {
 }
 
 func makeCreatePod(client clientset.Interface, namespace string, podTemplate *v1.Pod) error {
-	var err error
```
Comment (liggitt) [inline test/utils/runners.go]: doesn't removing this cause a compile error on line 1003?

Comment (jianglingxia) [reply test/utils/runners.go]: maybe not ,but if want left the line, i think line 997 to change like this :
if _, err = client.Core().Pods(namespace).Create(podTemplate); err == nil {
like 900 line,what about you? @liggitt  thanks

**Comment by jianglingxia** [comment]
PTAL @liggitt
thanks for your reply

**Comment by jianglingxia** [comment]
PTAL @liggitt
thanks

File: test/utils/runners.go (hunk 2 above)
Comment (liggitt) [reply test/utils/runners.go]: Yes, put this back as it was and change line 900

**Comment by jianglingxia** [comment]
I have updated it,for your approve @liggitt 
thanks

File: pkg/apis/apps/types.go [hunk 3]
```diff
@@ -61,6 +61,18 @@ const (
 	// replica count is changed, and will not wait for pods to be ready or complete
 	// termination.
 	ParallelPodManagement = "Parallel"
+	// RollingUpdateStatefulSetStrategyType indicates that update will be
```
Comment (liggitt) [inline pkg/apis/apps/types.go]: please put these consts back where they were... they are actually supposed to be of type StatefulSetUpdateStrategyType, which is a different issue, but putting them here is confusing and incorrect

**Comment by liggitt** [comment]
changes to the test file look fine. revert the const changes and squash

**Pull request closed without merging**